### PR TITLE
Release/0.2.0

### PR DESCRIPTION
**Pull request by mscwilson**
We are pleased to announce the release of Flutter tracker v0.2.0. It now works with Flutter 3!

We've updated the versions of the internal Snowplow trackers: it now uses JavaScript tracker v3.5 (for Web), Android v4, and iOS v4. The upgrade to JavaScript tracker v3.5 meant we were able to remove a JS file, making the tracker load more easily, and most importantly, load using Flutter 3.

The ability to set a custom POST path has also been added. Thanks to @erickok for the suggestion.

**CHANGELOG**

**New Features**
Configure custom POST path (#15)

**Bug fixes**
Remove loading custom JavaScript for session context and reading cookies (#13)

**Under The Hood**
Upgrade underlying mobile native trackers to version 4 (#17)
Fix schema link in documentation for ScreenView (#12)
Upgrade min Flutter, Dart and Android SDK versions and upgrade dependencies (#19) - thanks @koga for the Android upgrade!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +0,0 @@
-# 0.1.0
```
Comment (matus-tomlein) [inline CHANGELOG.md:1]: I understand that this changelog has a different format than the changelog in our other trackers but this is the format that is used in Dart packages and I think it is necessary for pub.dev to create a nice [Changelog section](https://pub.dev/packages/snowplow_tracker/changelog).

Question: Why did you remove the alpha releases from the changelog? They were also published on pub.dev so I think it makes sense to have them here.

**Review by matus-tomlein — APPROVED** [review]
🚀 🚢 

I had a couple of comments that I think should be addressed but I'm sure you'll handle that so approving already!

File: CHANGELOG.md (hunk 1 above)
Comment (mscwilson) [reply CHANGELOG.md:1]: Oh ok! I removed them as part of standardising with other changelogs. Will revert.

**Comment by mscwilson** [comment]
As soon as I get our release action working again 😅 hopefully today

**Comment by mscwilson** [comment]
@erickok It's released 😊

**Comment by erickok** [comment]
@mscwilson Thank you very much! It is working very well so far, including the new custom post path. Will report if we find any glitches whatsoever.

**Pull request merged**